code/obj/mining.dm

```diff
@@ -378,8 +378,28 @@
 				do_malfunction()
 			sleep(sleep_time)
 
-			var/datum/mining_encounter/MC = get_encounter(rarity_mod)
```

Comment:
doesn't look like to me anything calls `get_encounter` anymore?

Reply:
either remove it, or change the stuff to use it

Reply:
lines 421 and 422 look really fucked up too

Reply:
Does the small magnet exist in game? That whole code block (419-422) may not be needed if not.

Reply:
The get_encounter is still needed for line 421, so I can't delete lines 346 347 without some change there.

Reply:
no, like, look at the definition of `get_encounter`, it's literally `mining_controls.select_encounter(rarity_mod)` which you use twice. either use it or get rid of the helper proc

Reply:
I see, thank you. I have removed the get_encounter proc and the override under construction/small.